.github/pull_request_assignment.yml

```diff
@@ -103,108 +95,24 @@
     paths:
       - "specification/network/**"
     reviewers:
-      - JackTn
-      - keryul
+      - rkmanda
     branches:
       - release-network-Microsoft.Network-official
 
 - rule:
     paths: "specification/sql/**"
     reviewers:
```

Comment:
is this for both control and data plane specs?

Reply:
@weshaggard I think the directory is there?

- https://github.com/Azure/azure-rest-api-specs/tree/main/specification/sql/resource-manager

Reply:
I must have missed it. It is only mgmt though so we can remove it similar to the others.

Reply:
ok lets remove if redundant

Reply:
@weshaggard  @konrad-jamrozik , if assignments bot support other mechanisms of customized branches, then these rules can be removed.